packages/grid/_modules_/grid/models/api/gridStateApi.ts

```diff
@@ -11,7 +11,7 @@ export interface GridStateApi {
    * @param {string} stateId The part of the state to be returned.
    * @returns {any} The whole state or part of it.
```

Comment:
We also need to remove the "If `stateId` is present, only the referred part is returned." from the main description. 

```suggestion
   * @returns {any} The whole state.
```

--- 

I'll fix the issue with the docs generator in another PR. We might have other cases where we miss the generics.